File: 3367-find-the-sum-of-encrypted-integers/find-the-sum-of-encrypted-integers.py
class Solution:
    def sumOfEncryptedInt(self, nums: List[int]) -> int:
        ans = 0
        for num in nums:
            if num < 10:
                ans+=num
            else:
                s = str(num)
                m = max(int(i) for i in s)
                n = int(len(s)*str(m))
                ans+=n
        return ans

File: 3367-find-the-sum-of-encrypted-integers/find-the-sum-of-encrypted-integers.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var sumOfEncryptedInt = function(nums) {
    function encrypt(num){
        s = num.toString()
        m = Math.max(...s.split("").map((n)=>Number(n)))
        return Number((m.toString()).repeat(s.length))
    }
    let ans = 0
    for(let num of nums){
        if(num < 10){
            ans+=num
        }
        else{
            ans+=encrypt(num)
        }
    }
    return ans
};